qemu/tests/cfg/balloon_memhp.cfg

```diff
@@ -7,7 +7,7 @@
     login_timeout = 600
     no Host_RHEL.m6
     no RHEL.5
-    only Linux Win2008.x86_64 Win2012.x86_64 Win2016.x86_64 Win2019.x86_64
+    only Linux Win2008 Win2012 Win2016 Win2019 Win2022
```

Comment:
Suggest change to `no ***`,  then it can work in the future:  `no WinXP WinVista Win2000 Win2003 Win7 Win8 Win10`

For windows guest, does this case works for server only ?  As far as i know,  desktop version will keep use Win10, and use different minor release, such as win10-20h2, win10-21h1, so we don't need to worry about that i think.

Reply:
No new desktop as i mentioned in the pre comment,  it will always be Win10. This is not a problem.

Reply:
When a new supported version come out, that is test plan's response,  no need to update auto code then.

Reply:
Thanks, updated.